include/loop-analysis/sparse-analysis.hpp

```diff
@@ -32,6 +32,7 @@
 #include "tiling-tile-info.hpp"
 #include "model/sparse-optimization-info.hpp"
 #include "model/topology.hpp"
+#include <math.h>
```

Comment:
C library includes should preferably be included using, e.g., <cmath> instead of <math.h>.
Also, system-wide library includes should precede the block of local includes (i.e., in this case cmath should come before the local .hpp includes)